### ISSUE TITLE
Fix known issues on new site

### ISSUE THREAD

**Issue by bitcoinhodler** [GlacierProtocol/glacierprotocol.github.io#7]
The following issues are still unresolved from the test site. Filing here so they are not forgotten:

https://github.com/joaofnfernandes/glacierprotocol.github.io/issues/8 (despite being marked as closed)
https://github.com/joaofnfernandes/glacierprotocol.github.io/issues/11 (despite being marked as closed)
https://github.com/joaofnfernandes/glacierprotocol.github.io/issues/12
https://github.com/joaofnfernandes/glacierprotocol.github.io/issues/14

**Comment by diogomonica** [comment]
Thanks for filling these @bitcoinhodler 

@joaofnfernandes what do think? We wanted to simplify/change some portions of the protocol, not sure if any of these fall into the portions we discussed before.

**Comment by joaofnfernandes** [comment]
Here's the general direction I think we should take to make the protocol more accessible to new users:
* Clean up and expand the [overview section](https://glacierprotocol.org/docs/overview/), so that we can explain in a central place what the protocol is about and explain some terminology
* The design doc is still in PDF, and we should integrate it into the new website. Once that is done we can easily link between the protocol and design doc, and this will help people assess risk and better understand certain concepts.
* Once that is done, we can clean the actual protocol steps to be more direct and link back to the common terminology page
* Clean up the protocol steps to that they are easier to read. Also explore if we can use checkboxes instead of list items, so that users can mark things as they run through each step.
* Add a contributor's guide so that others know how to do with, and have a styleguide for people to follow.
* Add some screenshots to better guide users - joaofnfernandes#12.

**Comment by cocomorris** [comment]
Hi, I found this typo at the very bottom of the page, could be confusing. 

https://glacierprotocol.org/docs/deposit/generate-cold-storage-data/

> If there are any discrepancies, do not proceed.
>...  If they are different by more than 3 characters, restart the Deposit Protocol.
>        If the are identical, restart the Deposit Protocol.

This should read "If they are not are identical, restart the Deposit Protocol."

**Comment by bitcoinhodler** [comment]
> 
> This should read "If they are not are identical, restart the Deposit Protocol."

This is incorrect. The context here is that the two quarantined laptops have generated different keys from (what should be) the same entropy. There are three possibilities outlined: (i) different by 1-3 characters; (ii) different by more than 3 characters; and (iii) identical.

Possibility (i) is due to a simple transcription error and can be safely corrected. Possibilities (ii) and (iii) are not explained so easily, so the solution is to restart the entire protocol.

The case where they are *not* identical is already covered by the combination of (i) and (ii).

Though we should fix "the" to "they".

This is also [GlacierProtocol#7](https://github.com/GlacierProtocol/GlacierProtocol/issues/7).
